src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -372,7 +372,6 @@ static inline int MPIDI_OFI_do_put(const void *origin_addr,
                               rdma_inject_write);
         goto fn_exit;
     } else if (origin_contig && target_contig) {
-        MPIR_ERR_CHKANDJUMP((origin_bytes != target_bytes), mpi_errno, MPI_ERR_SIZE, "**rmasize");
         MPIDI_OFI_INIT_SIGNAL_REQUEST(win, sigreq, &flags);
```

Comment:
Why do you delete it ? Is it a fix for `do_put` or something should be put into the code refactoring commit ?